dnsapi/dns_nsupdate.sh

```diff
@@ -39,13 +39,15 @@ dns_nsupdate_add() {
   [ -n "$DEBUG" ] && [ "$DEBUG" -ge "$DEBUG_LEVEL_1" ] && nsdebug="-d"
   [ -n "$DEBUG" ] && [ "$DEBUG" -ge "$DEBUG_LEVEL_2" ] && nsdebug="-D"
   if [ -z "${NSUPDATE_ZONE}" ]; then
-    nsupdate -k "${NSUPDATE_KEY}" $nsdebug "${NSUPDATE_OPT}" <<EOF
+    #shellcheck disable=all
```

Comment:
don't disable all,  why not just disable the specific number?

Reply:
@Neilpang Fixed in https://github.com/acmesh-official/acme.sh/commit/9ecd84080b5f5160626de7c886b30831144efbc9.